SourceX/miniwin/misc.cpp

```diff
@@ -190,6 +190,14 @@ bool SpawnWindow(LPCSTR lpWindowName, int nWidth, int nHeight)
 		if (SDL_RenderSetLogicalSize(renderer, nWidth, nHeight) <= -1) {
 			ErrSdl();
 		}
+#endif
+	} else {
+#ifdef USE_SDL1
+		refreshDelay = 16; // rougly 60hz
+#else
+		SDL_DisplayMode mode;
+		SDL_GetDisplayMode(0, 0, &mode);

```

Comment:
This query first monitor, technically there can be several but then we would also have to figure out what monitor we are running on, and the user may change that at any time. So given that `!upscal` is not seen as the primer target I think this as an acceptable solution.